aligner/__main__.py

```diff
@@ -51,8 +51,8 @@
                            description="Prosodylab-Aligner")
 argparser.add_argument("-c", "--configuration",
                        help="config file")
-argparser.add_argument("-d", "--dictionary", default=DICTIONARY,
-                       help="dictionary file (default: {})".format(DICTIONARY))
+argparser.add_argument("-d", "--dictionary", metavar="DICT", dest="dictionaries", action="append",
```

Comment:
I understand why you did it this way but I think there is a clearer way that gets rid of the hacky bit on lines 85-86 and lines 149-150: use `nargs="+"` instead. If you get rid of the added stuff here and add `nargs="+"` then you can specify one or more arguments after `-d` (note that you don't say `-d d1 -d d2 ...` but rather `-d d1 d2 ...`). Then you don't need to make this a list and you don't need any global state (which is what `metavar` and `dest` are, really.)

What do you think?

Reply:
The reason why I avoided this approach is because the `-d DICT1 DICT2` command-line syntax is quite unusual. This is meant to be "the `-d` option is taking two arguments, `DICT1` and `DICT2`," but the more common interpretation is "the `-d` option is taking a single `DICT1` argument, and `DICT2` is being passed as a non-option argument." I don't know of any common command-line program that uses the former interpretation.

(If you insist on this, I can do it, but my desire here is to have the command-line interface conform to established conventions. I've already been working on a later PR that reorganizes the options to be less confusing.)

The hackish bit is actually due to a different issue, specifically [this](https://bugs.python.org/issue16399) Python foible. (Without that, there would be no way to get `eng.dict` out of the list of dictionaries being passed in.)

Not sure what you mean by "global state"; the `metavar` and `dest` args are just overriding the defaults to something more sensible...